src/lib.rs

```diff
@@ -370,24 +370,30 @@ fn encode_block(fi: &FrameInvariants, fs: &mut FrameState, cw: &mut ContextWrite
     }
 
     let tx_type = TxType::DCT_DCT;
+    let tx_size = match bsize {
+        BlockSize::BLOCK_8X8 => TxSize::TX_8X8,
+        _ => TxSize::TX_4X4
```

Comment:
What happens to the chorma block (which is 4x4) of BLOCK_8X8 luma?
Cant use 8x8 tx for 4x4 chroma!

Reply:
See let uv_tx_size later. It was broken though, so I fixed it.